Build failure with servant 0.16

```
> /tmp/stackage-build13$ stack unpack mixpanel-client-0.1.1@sha256:80d4c717d76686ca770cfebaf45745219e9d33ac9da6aaa946033e71e49c4d23
Unpacked mixpanel-client-0.1.1 to /tmp/stackage-build13/mixpanel-client-0.1.1/
> /tmp/stackage-build13/mixpanel-client-0.1.1$ ghc -clear-package-db -global-package-db -package-db=/var/stackage/work/builds/nightly/pkgdb Setup
[1 of 1] Compiling Main             ( Setup.hs, Setup.o )
Linking Setup ...
> /tmp/stackage-build13/mixpanel-client-0.1.1$ ./Setup configure --package-db=clear --package-db=global --package-db=/var/stackage/work/builds/nightly/pkgdb --libdir=/var/stackage/work/builds/nightly/lib --bindir=/var/stackage/work/builds/nightly/bin --datadir=/var/stackage/work/builds/nightly/share --libexecdir=/var/stackage/work/builds/nightly/libexec --sysconfdir=/var/stackage/work/builds/nightly/etc --docdir=/var/stackage/work/builds/nightly/doc/mixpanel-client-0.1.1 --htmldir=/var/stackage/work/builds/nightly/doc/mixpanel-client-0.1.1 --haddockdir=/var/stackage/work/builds/nightly/doc/mixpanel-client-0.1.1
Configuring mixpanel-client-0.1.1...
> /tmp/stackage-build13/mixpanel-client-0.1.1$ ghc -clear-package-db -global-package-db -package-db=/var/stackage/work/builds/nightly/pkgdb Setup
> /tmp/stackage-build13/mixpanel-client-0.1.1$ ./Setup build
Preprocessing library for mixpanel-client-0.1.1..
Building library for mixpanel-client-0.1.1..
[1 of 5] Compiling MixPanel.Types.Core ( src/MixPanel/Types/Core.hs, dist/build/MixPanel/Types/Core.o )
[2 of 5] Compiling MixPanel.Types.EngageData ( src/MixPanel/Types/EngageData.hs, dist/build/MixPanel/Types/EngageData.o )
[3 of 5] Compiling MixPanel.Types.TrackData ( src/MixPanel/Types/TrackData.hs, dist/build/MixPanel/Types/TrackData.o )
[4 of 5] Compiling MixPanel.API     ( src/MixPanel/API.hs, dist/build/MixPanel/API.o )
[5 of 5] Compiling MixPanel         ( src/MixPanel.hs, dist/build/MixPanel.o )

src/MixPanel.hs:42:18: error:
    Not in scope: type constructor or class ‘ServantError’
    A data constructor of that name is in scope; did you mean DataKinds?
   |
42 |   = ServantError ServantError
   |                  ^^^^^^^^^^^^
```

--- a/src/MixPanel.hs
+++ b/src/MixPanel.hs
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP #-}
 module MixPanel
   ( MixPanelError(..)
   , AuthToken(..)
@@ -34,12 +35,15 @@ import           MixPanel.Types.TrackData       ( TrackData(..)
                                                 )
 import           MixPanel.Types.EngageData      ( EngageData, DistinctId, Operation(..), mkEngageData )
 
+#if !MIN_VERSION_servant_client(0,16,0)
+#define ClientError ServantError
+#endif
 
 host :: BaseUrl
 host = BaseUrl Https "api.mixpanel.com" 443 ""
 
 data MixPanelError
-  = ServantError ServantError
+  = ClientError ClientError
   | Error Text
   deriving (Show, Exception)
 
@@ -71,7 +75,7 @@ runMixPanel :: ClientEnv -> ClientM DidSucceed -> IO (Either MixPanelError ())
 runMixPanel clientEnv comp = do
   result <- runClientM comp clientEnv
   return $ case result of
-    Left err -> Left $ ServantError err
+    Left err -> Left $ ClientError err
     Right (Fail err) -> Left $ Error err
     Right Success -> Right ()
 

--- a/stack.yaml
+++ b/stack.yaml
@@ -1,4 +1,4 @@
-resolver: lts-12.19
+resolver: lts-13.9
 packages:
 - .
 nix:

